### PR TITLE
[RFC] Renaming, The Great

### PR DESCRIPTION
**Pull request by rustyrussell**
This of course is very fragile, as almost any change will break it.

### CODE REVIEW

File: lightningd/peer_control.c [hunk 1]
```diff
@@ -259,11 +259,11 @@ resolve_close_command(struct lightningd *ld, struct channel *channel,
 /* Destroy the close command structure in reaction to the
  * channel being destroyed. */
 static void
-destroy_close_command_on_channel_destroy(struct channel *_ UNUSED,
+close_command_on_channel_destroy_destroy(struct channel *_ UNUSED,
```
[the remark this answers is not in the document]
Comment (cdecker) [reply lightningd/peer_control.c]: Could flip it around: `close_command_destroy_on_channel_destroy`, i.e., `close_command` is the namespace, `destroy` the action, and `on_channel_destroy` the event we're reacting to.

**Comment by cdecker** [comment]
Happy to drop consistency in this case, I wasn't going to propose changing everything anyway, might just be something to keep in mind for future code :wink:

**Comment by niftynei** [comment]
Hmm. Curse the incompatibility of semantics and consistency. Given that the first is emotional and the second rational, I'd argue in favor of the emotional argument (ie for semantics and therefore not merging this), as it'll have more bearing on how I feel about the codebase on a day-to-day / practical basis.

**Comment by cdecker** [comment]
Ok, so we have 2 votes against, and my vote for this, so I'll close this in favor of less brain damage in the future. I hope you're ok if I stick with my scheme for stuff that I write :wink:

**Pull request closed without merging**